terraform/modules/networking/subnet.tf

```diff
@@ -0,0 +1,49 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  nat_gateway_cidr_10 = cidrsubnet(aws_vpc.main.cidr_block, 6, 0) # 0.0/22 - public
+  public_cidr_10      = cidrsubnet(aws_vpc.main.cidr_block, 6, 1) # 4.0/22 - for alb listeners
+  firewall_cidr_10    = cidrsubnet(aws_vpc.main.cidr_block, 6, 2) # 8.0/22
+  private_cidr_10     = cidrsubnet(aws_vpc.main.cidr_block, 6, 3) # 12.0/22- ecs and bastion host
+  isolated_cidr_10    = cidrsubnet(aws_vpc.main.cidr_block, 6, 4) # 16.0/22
+}
+
+resource "aws_subnet" "nat_gateway" {
+  availability_zone = data.aws_availability_zones.available.names[0]
```

Comment:
Probably best to chat about this in case I've misunderstood something. We discussed RTO and RPO at TDA in the context of RDS only - and pitched the standard multi-az RDS setup - i.e. 1x writer instance in one AZ, with a passive standby in a second AZ (plus other cross-region backups etc). I'd inferred that we'd therefore only deploy the application to one AZ as well, but that might not follow.

Reply:
As with some other places, it feels sensible to have an ADR for this which references the agreed RTO and RPO, and disaster recovery plan, if not completely rehashing it :)